e2e-tests/spring-rpc/spring-rpc-thrift/src/test/java/org/evomaster/e2etests/spring/rpc/examples/fakemockobject/FakeMockObjectEMTest.java

```diff
@@ -43,12 +43,23 @@ public void testRunEM() throws Throwable {
                     args.add("--enableBasicAssertions");
                     args.add("true");
 
+//                    args.add("--saveScheduleTaskInvocationAsSeparatedFile");
+//                    args.add("true");
+
+                    args.add("--enableCustomizedMethodForScheduleTaskHandling");
```

Comment:
when adding new tests, or modifying existing ones, best to start use the new `setOption()`